app/dashboard/views.py

```diff
@@ -2046,23 +2046,23 @@ def profile(request, handle):
     context['sent_kudos_count'] = sent_kudos.count()
     context['verification'] = profile.get_my_verified_check
     context['avg_rating'] = profile.get_average_star_rating
-
+    
     context['unrated_funded_bounties'] = Bounty.objects.current().prefetch_related('fulfillments', 'interested', 'interested__profile', 'feedbacks') \
         .filter(
             bounty_owner_github_username__iexact=profile.handle,
             network=network,
         ).exclude(
             feedbacks__feedbackType='approver',
             feedbacks__sender_profile=profile,
-        )
+        ).distinct('pk')
```

Comment:
Doesn't need also a "current_bounty" to just take the last ?

Reply:
`.current()` takes care of that..

Reply:
cool I was the one adding current in some PR trying to solve this dupe 😄